packages/components/pie-button/test/visual/pie-button.spec.ts

```diff
@@ -14,18 +23,36 @@ const props: PropObject = {
     disabled: [true, false],
 };
 
-const componentPropsMatrix : Combination[] = getAllPropCombinations(props);
-const componentPropsMatrixByVariant: Record<string, Combination[]> = splitCombinationsByPropertyValue(componentPropsMatrix, 'variant');
+// Renders a <pie-button> HTML string with the given prop values
+const renderTestPieButton = (propVals: WebComponentPropValues) => `<pie-button variant="${propVals.variant}" size="${propVals.size}" type="${propVals.type}" ${propVals.isFullWidth ? 'isFullWidth' : ''} ${propVals.disabled ? 'disabled' : ''}>Hello world</pie-button>`;
```

Comment:
As this is using string interpolation, can it be written so that it's a bit easier to parse visually?

```suggestion
const renderTestPieButton = (propVals: WebComponentPropValues) => `
    <pie-button
        variant="${propVals.variant}"
        size="${propVals.size}"
        type="${propVals.type}"
        ${propVals.isFullWidth ? 'isFullWidth' : ''}
        ${propVals.disabled ? 'disabled' : ''}>
        Hello world
    </pie-button>`;
```